src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/IDispatchComObject.cs

```diff
@@ -230,7 +230,7 @@ internal bool TryGetMemberMethodExplicit(string name, out ComMethodDesc method)
                 return false;
             }
 
-            throw Error.CouldNotGetDispId(name, string.Format(CultureInfo.InvariantCulture, "0x{0:X})", hresult));
+            throw Error.CouldNotGetDispId(name, $"0x{(uint)hresult:X})");
```

Comment:
>) [](http://example.com/codeflow?start=69&length=1)

I know you didn't introduced this `)` but I am wondering if it is really intentional in this format? This comment apply to the other change in this file too.

Reply:
It's formatting the value as hex.  What do you think it should be doing instead?

Reply:
I meant it is including `)` at the end which I am not sure why.

Reply:
Ah, yeah, I didn't notice that paren.  That does look wrong.  But I'm not going to mess with it :)